skimage/filters/thresholding.py

```diff
@@ -1088,6 +1094,8 @@ def threshold_multiotsu(image, classes=3, nbins=256):
                                   nbins=nbins,
                                   source_range='image')
     prob = hist / image.size
+    # histogram ignores nbins for integer arrays.
+    nbins = len(bin_centers)
```

Comment:
why?

Reply:
Because of our [histogram implementation](https://github.com/scikit-image/scikit-image/blob/master/skimage/exposure/exposure.py#L93):
```
    nbins : int, optional
        Number of bins used to calculate histogram. This value is ignored for
        integer arrays.
```

Reply:
gah sorry, thanks! I was held up on merging this because I didn't fully understand why you used `len(bin_centers)` instead of `len(hist)` but thye are the same length.